components/atoms/Select/single-select.tsx

```diff
@@ -35,7 +35,14 @@ const SingleSelect = ({
 
   useEffect(() => {
     if (isOpen && isSearchable) {
-      inputRef.current?.focus();
+      // TODO: Revisit this. There's a timing issue where the input's ref is set when the dropdown is opened
+      // but it still can't receive focus. This is a workaround to focus the input after
+      // the dropdown is opened.
+      setTimeout(() => {
```

Comment:
As mentioned this is a workaround. I could dig further, but I wasted a lot of time on this already. Basically the `inputRef.current` is non-null but not when `isOpen` and `isSearchable` are true. I tried some things including the non-recommended putting `inputRef.current` in the dependency array but reverted one, because it's not recommmended and can cause unnecessary rerenders, but two, it didn't work. 🙃

Maybe there is something else going on wit the Command* components that I'm unaware of, but for now, I think this workaround it fine.